src/createAssetPackage.js

```diff
@@ -40,12 +40,18 @@ function getFileSuffixFromMimeType(mime) {
     return '.ico';
   }
 
-  const match = mime.match(/^image\/(.+)$/);
-  if (!match) {
-    return '';
+  const imgMatch = mime.match(/^image\/(.+)$/);
+  if (imgMatch) {
+    // other images
+    return `.${imgMatch[1]}`;
   }
-
-  return `.${match[1]}`;
+  if (mime.startsWith('text/html')) {
+    return '.html';
+  }
+  if (mime.startsWith('text/plain')) {
+    return '.txt';
+  }
```

Comment:
Are there others we should also do here, like .json maybe?

Perhaps we should use the mime-types package, which can look up extensions based on mime type: https://github.com/jshttp/mime-types?tab=readme-ov-file#mimeextensiontype